java/src/jmri/jmrit/display/layoutEditor/TrackSegment.java

```diff
@@ -1,15 +1,14 @@
 package jmri.jmrit.display.layoutEditor;
 
-import static jmri.jmrit.display.layoutEditor.LayoutTrack.NONE;
 import static jmri.util.MathUtil.add;
```

Comment:
Just playing devils advocate here… but I think that this:
`offset = MathUtil.multiply(MathUtil.normalize(offset), MathUtil.length(offset) / 3);`
is harder to read than: `offset = multiply(normalize(offset), length(offset) / 3);`.  This `g2.draw(new Line2D.Double(center, MathUtil.oneThirdPoint(center, MathUtil.midPoint(pointA, pointB))));` is unnecessarily verbose when it could be: `g2.draw(new Line2D.Double(center, oneThirdPoint(center, midPoint(pointA, pointB))));`. (Same for `math.fabs`.) In netBeans I can always command-click on the method name to goto the implementation… and be specific if it's the wrong one.
All that said… I do understand that others may not be using netBeans and not have the ability to easily find an implementation… So for them I will (humbly?) comply… ;-)